### ISSUE TITLE
Fix potential bug in .vi file section reading code

### ISSUE THREAD

**Issue by delphidabbler** [delphidabbler/vied#78]
The keys within a section are case sensitive, but the dictionary used to store them has an ambiguous comparer object. The Hash function is case sensitive, but the equality function is not:

```pascal
  fEntries := TDictionary<string,string>.Create(
    TDelegatedEqualityComparer<string>.Create(
      function(const Left, Right: string): Boolean
      begin
        Result := SameText(Left, Right, loInvariantLocale);
      end,
      function(const Value: string): Integer
      begin
        Result := ElfHash(Value);
      end
    )
  );
```

To ensure consistency the equality function should changed to be case sensitive:

```pascal
      function(const Left, Right: string): Boolean
      begin
        Result := SameStr(Left, Right, loInvariantLocale);
      end,
```

**Comment by delphidabbler** [comment]
Forgot to say that this code is in `UVIData.TVIDataSection.Create`

**Comment by delphidabbler** [comment]
Implemented at commit f60db10
